eth/protocols/eth/broadcast.go

```diff
@@ -142,6 +142,12 @@ func (p *Peer) announceTransactions() {
 		fail   = make(chan error, 1) // Channel used to receive network error
 		failed bool                  // Flag whether a send failed, discard everything onward
 	)
+
+	queueLimit := maxQueuedTxAnns
+	if p.IsTrusted() || p.IsStatic() {
+		queueLimit = maxQueuedTxAnnsTrusted
```

Comment:
Should we have just considered 10x maxQueuedTxAnns rather than defining another const?